docs/settings.rst

```diff
@@ -94,6 +94,7 @@ Setting name (followed by default value, if any)
                                                                                  control data. For example: ``[".hg", ".git", ".bzr"]``
 ``JINJA_EXTENSIONS = []``                                                        A list of any Jinja2 extensions you want to use.
 ``JINJA_FILTERS = {}``                                                           A dictionary of custom Jinja2 filters you want to use.
+``JINJA_TESTS = {}``                                                             A dictionary of custom Jinja2 tests you want to use
```

Comment:
Adding a link to http://jinja.pocoo.org/docs/dev/api/#writing-tests can help people getting started with Jinja2 tests.